src/pybind/mgr/dashboard/frontend/package.json

```diff
@@ -33,6 +33,8 @@
     "ng2-charts": "^1.6.0",
     "ng2-toastr": "4.1.2",
     "ngx-bootstrap": "^2.0.1",
+    "patternfly": "^3.52.3",
+    "rcue": "^3.48.1",
```

Comment:
It is not necessary to explicitly set "patternfly" as a dependency as "rcue" has it already as a dependency. In fact, "rcue" depends on an earlier version of patternfly:
{
  "name": "rcue",
  "version": "3.48.1",
  "author": "Red Hat",
  "license": "Apache-2.0",
  "dependencies": {
    "bootstrap": "~3.3.7",
    "font-awesome": "^4.7.0",
    "jquery": "~3.2.1",
    "patternfly": "3.48.1"
}

Reply:
Ack @alfonsomthd, I will remove patternfly dependency.